guide/payments/introduction.md

```diff
@@ -0,0 +1,86 @@
+
+> **Getting Started / Why bitcoin is unique**
+>
+> The economic definition of money typically involves three specific [functions](https://en.wikipedia.org/wiki/Money#Functions) (medium of exchange, unit of account, store of value) and five [properties](https://en.wikipedia.org/wiki/Money#Properties) (durability, portability, fungibility, scarcity, divisibility, recognizability). Although bitcoin is a new breed of money, it still satisfies all of these requirements. Instead of relying on physical properties (like gold and silver) or trust in central authorities (like traditional currencies), bitcoin relies on a network of thousands of computers to enforce its mathematically defined ruleset refered to as the protocol.
+
+## Bitcoin is Global
+
+We are for designing the next evolution of money. One which is internet based, and global.
+
+> **Getting Started / Why bitcoin is unique**
+>
+> Bitcoin can be used by anyone with an internet connection, which is available to around [53.6% of the world population](https://en.wikipedia.org/wiki/Global_Internet_usage), a number that is growing quickly. Some aspects of bitcoin, like key and address generation, even work without being online. This makes it the first truly globally available form of money that is unbound by country borders. People from around the world can freely transact without having to rely on middlemen like currency exchanges and local payment services.
+
+## Bitcoin is extensible, it's not just a payment network
+
+It should be noted, that bitcoin is not just a mechanism to make simple payments but it is a complete global financial infrasutructure for moving value without intermediaries. Each transaction is like a bit of code, which means it can be programmed — for example you can set various spending conditions like requiring multiple parties to agree prior funds being able to move. Its even possible to restrict funds from being moved until a certain time.
+
+## Bitcoin is secure and accessible to all
+
+Some people may have the perception that bitcoin is anonymous. This isn't quite accurate because all transactions are public so anyone can see the entire history of transfers that ever happened going back to the very first one on [2009-01-03](https://blockstream.info/tx/4a5e1e4baab89f3a32518a88c31bc87f618f76673e2cc77ab2127b7afdeda33b).
+
+#### Why are transactions public?
+
+They are public so that anyone can have the ability to verify the entire history of transactions themselves. We call this a permissionless system. No one needs to ask for permission to send a transaction or to build experiences on top of bitcoin.
+
+#### What about privacy?
+
+Even though all transactions are stored publicly, there is no personally identifiable information attached to them. Privacy is still super important though since every time you make a payment the person receiving can see where you funded that transaction from, as you would be able to trace which address they spend those funds.
```

Comment:
A suggestion but you may want to break this lengthy sentence down into a few shorter ones to make it easier to follow who's doing what.
```suggestion
Even though all transactions are stored publicly, there is no personally identifiable information attached to them. Privacy is still important though since every time you make a payment, the person receiving can see where you funded that transaction from, as you would be able to trace which address they spend those funds.
```

Reply:
I am starting to think that we need a section that explains this more. The fact that I can see the history of your bitcoin, does not mean I can make sense of it. How do I know who is behind each address? It takes a huge effort to deanonymize addresses, so I think it might be helpful to clarify this so people are not overly paranoid.

I'd also be interested in some more data in how effective privacy-preserving techniques are. Does avoiding address re-use make me 99% safe from chain analysis? Or is mixing to an anonymity set of 50 the standard?

Reply:
Unfortunately even with carefully selecting UTXOs to spend, you may still end up using those at an exchange which is KYCed. Or someone else may doxx one of your addresses you shared with them by publishing it online. So there's so silver bullet for privacy, it's also something thats not really quantifiable.

Also, it is worth making people more paranoid in my opinion. The mere fact that we struggle with it currently and we are in the know, interacting with more advanced tools than the wider population is aware of means that we're taking autonomy from the users.

Reply:
@GBKS I was also thinking some of this content can also live in the Getting Started. I'm fine with moving some more generally applicable parts over there and link over to it.